decred.go

```diff
@@ -96,6 +96,9 @@ func (c *crawler) testPeer(ctx context.Context, ip netip.AddrPort) {
 	// Wait for the verack message or timeout in case of failure.
 	select {
 	case <-verack:
+		if p.ProtocolVersion() < wire.RemoveRejectVersion {
```

Comment:
Seems OK. This protocol version was released in 1.7.0, nodes running code older than that are forked off the network and not worth connecting to.